extensions/amp-next-page/amp-next-page.md

```diff
@@ -25,134 +25,119 @@ limitations under the License.
   </tr>
   <tr>
-    <td><strong>Examples</strong></td>
+    <td width="40%"><strong>Examples</strong></td>
     <td>See AMP By Example's <a href="https://amp.dev/documentation/examples/components/amp-next-page/">amp-next-page example</a>.</td>
   </tr>
 </table>
 
-[TOC]
+## Version notes
 
-## Behavior
+| Version | Description                                                                                                                                                                                                                                                                          |
+| ------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| 1.0     | Support for an infinite number of page recommendations, reduced bundle size, improved API, support for amp-analytics, templated separators and footers, better handling of fixed elements.<br><br>API changes are breaking, please take a look at the migration section for details. |
+| 0.1     | Initial experimental implementation. Limited to three recommended documents                                                                                                                                                                                                          |
+
+## Usage
 
 Given a list of pages, `amp-next-page` tries to load them after the current
-document, providing an infinite-scroll type experience.
+document, providing an infinite-scroll experience.
 
 Pages should be specified in a JSON config loaded from a remote URL listed in
-the element `src`, or inlined as a `script` element child.
+the element `src`, or inlined as a `script` element child (or both).
 
 ```html
 <amp-next-page src="https://example.com/next-page-config.json"></amp-next-page>
 <!-- Or -->
 <amp-next-page>
   <script type="application/json">
-    {
-      "pages": ...
-    }
+    [
+      {
+        "url": ...,
+        "title": ...,
+        "image": ...
+      },
+      ...
+    ]
   </script>
 </amp-next-page>
 ```
 
 If loading the next document is successful it will be appended to the end of
-the current document as a child of the `amp-next-page` component, moving any
-content after it further down the page. Because of this the component should
-usually be placed directly after the unique content of a given page: for
-example, at the end of a news article or recipe, but before the footer or
-other content repeated across articles.
+the current document as a child of the `amp-next-page` component. Because of this the component must be the last child of the body element, for
+example, at the end of a news article or recipe. If needed, any footer content should be embedded inside the `<amp-next-page>` tag and will be displayed once no more article suggestions are available.
```

Comment:
Trying to figure out where `If needed, any footer content should be embedded inside the <amp-next-page> tag and will be displayed once no more article suggestions are available.` information best belongs. Question:

Is this type of footer different from the "Footer recommendation box" as in, it's a "classic" page footer?

Reply:
^ That's correct, this is the actual site footer